app/views/layouts/_shim.html.erb

```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js">
+  </script>
+<![endif]-->
```

Comment:
file  này comment hết thì còn gì đâu e?

Reply:
comment này để load js khi IE version < 9
link em tham khảo: https://stackoverflow.com/questions/52492279/how-to-only-insert-a-script-for-browsers-above-ie-9

Reply:
oke em. Chừa line trống cuối file này nhé